RTech.CoffeeMachine.Api.Tests/BrewCoffee/Controllers/CoffeeMachineControllerTests.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System.Net;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.AspNetCore.TestHost;
+using Microsoft.Extensions.Caching.Memory;
+using Moq;
+using RTech.CoffeeMachine.Api.BrewCoffee.Controllers;
+using RTech.CoffeeMachine.Api.BrewCoffee.Models;
+using RTech.CoffeeMachine.Api.BrewCoffee.Util;
+using Shouldly;
+
+namespace RTech.CoffeeMachine.Api.Tests.BrewCoffee.Controllers;
+
+public class CoffeeMachineControllerTests : IClassFixture<WebApplicationFactory<Program>>
```

Comment:
This is the controller with the new endpoint.
Note that it is in a vertical slice architecture (most new code goes into a new root folder named `BrewCoffee`)